db.tf

```diff
@@ -68,3 +69,37 @@ resource "aws_route53_record" "mysql-shared-dns" {
   ttl = 300
   records = ["${aws_db_instance.mysql-shared-db.address}"]
 }
+
+resource "aws_db_instance" "postgres-shared-db" {
```

Comment:
Something I forgot to do with the MySQL instance was give it an `identifier`, which is the name that shows up in the console. Use `postgres-shared-db` as with the terraform resource name.